src/main/java/spotify/factories/AbstractPlayableObjectDeserializer.java

```diff
@@ -7,7 +7,23 @@
 
 import java.lang.reflect.Type;
 
+/**
+ * This class is a custom {@link JsonDeserializer} which deserializes json and returns a {@link AbstractPlayableObject} subclass based on the type property in the json
+ * @see spotify.models.generic.AbstractPlayableObject
+ * @author Jiankai Zheng (jk.zheng@hotmail.com)
+ * @since // check with Jiankai
```

Comment:
should not be committed if it's not finished, if you have a question about it, you can place a comment when creating the pull request.

Reply:
Thanks. For next PR, I will ask you questions after I finish them all.